src/main/scala/Utils.scala

```diff
@@ -0,0 +1,19 @@
+
+import com.typesafe.scalalogging.LazyLogging
+import com.codahale.metrics.{Metric, MetricFilter}
+import scala.util.matching.Regex
+
+object Utils extends LazyLogging {
+    def buildMetricFilter(filterRegexPattern: String): MetricFilter = {
+        //since matches has is a string, the whitelist should be log filenames with commas with spaces
+        return new MetricFilter() {
+            override def matches(string: String, metric: Metric): Boolean =  {
+                logger.debug("Metric Name:" + string)
```

Comment:
This debug log doesn't really help with anything, since it just adds a message with a random metric name